setup/context/commands_test.go

```diff
@@ -0,0 +1,69 @@
+package context
```

Comment:
I'm pretty hesitant to give the :+1:  on these unit tests and introducing the whole `FileIO` interface for a couple of reasons:

* This tool isn't setup to be unit tested. In my opinion, adding more interfaces, classes, and mocks on top of the current architecture isn't a step in the right direction - it just adds more complexity and maintenance costs to an already complex tool. If we want unit tests, we should do it right or not at all. 
* Creating your own mock class (`MockIO`) is a pit we fell into when first starting go. The "better" way of generating mocks is is via a tool called [gomock](github.com/golang/mock). That being said, regardless of how the mock class is generated/updated, the larger issue I have with it is the additional maintenance cost. 
* This is echoing the first point a bit - but these unit tests don't really _add_ anything. They essentially just check if `json.Marshal` works. If having unit tests don't give you confidence in your code - why have them at all? Don't get me wrong - I'm not trying to say that Jonathan writes bad unit tests - what I **am** saying is that the current architecture is so bad that it's damn near impossible for anyone to write a useful unit test. 

TLDR: I think it would be better to drop the unit tests and associated interfaces/mocks/etc from this PR in favor of re-architecting this tool to be more unit testable.